tests/versioned/apollo-server/transaction-naming.test.js

```diff
@@ -10,7 +10,7 @@ const semver = require('semver')
 
 const { afterEach, setupCoreTest } = require('../../lib/test-tools')
 
-const transactionNamingTests = require('../transaction-naming-tests')
+const transactionNamingTests = require('./transaction-naming-tests')
```

Comment:
i don't see the need for this file. you can prob just move the `transaction-naming-tests` into here or make that file run the tests.